jwt/api.py

```diff
@@ -25,6 +25,25 @@ def __init__(self, algorithms=None):
             if key not in self._valid_algs:
                 del self._algorithms[key]
 
+        if not options:
+            options = {}
+
+        self.default_options = {
+            'verify_signature': True,
+            'verify_exp': True,
+            'verify_nbf': True,
+            'verify_iat': True,
+            'verify_aud': True,
```

Comment:
they should at least be labeled with expansions, or have a "constants" class that labels the keys, such as:

    class CLAIMS_KEYS(object):
        class VERIFY(object):
            EXPIRATION = 'verify_exp'
            ISSUED_AT = 'verify_iat'